connector/http-agent/app/Utils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * A module with helper functions
+ * @module utils
+ */
+
+/**
+ * @function parseTimestamp
+ *
+ * Return timestamp in proper format
+ *
+ * @param {string} ts
+ *
+ * @returns {number}
+ */
+const parseTimestamp = (ts) => {
+  if (ts) {
```

Comment:
This validation only checks if there is a value, but not if the value is a valid date, it can lead to a code break.

Reply:
Done.

Reply:
In this case, try catch won't work because the Date.parse function doesn't throw an exception, it returns a value that can be NaN or "Invalid Date"

https://docs.w3cub.com/javascript/global_objects/date/parse